nifi-commons/nifi-properties/src/main/java/org/apache/nifi/util/NiFiProperties.java

```diff
@@ -81,6 +81,7 @@
     public static final String PROCESSOR_SCHEDULING_TIMEOUT = "nifi.processor.scheduling.timeout";
     public static final String BACKPRESSURE_COUNT = "nifi.queue.backpressure.count";
     public static final String BACKPRESSURE_SIZE = "nifi.queue.backpressure.size";
+    public static final String BOOTSTRAP_PORT = "nifi.bootstrap.port";
```

Comment:
Could you rename this to nifi listener bootstrap port as well to keep in sync?

Reply:
Thanks for reviewing. THis is been fixed, squashed, and rebased to the latest master.